tests/python/unittest/test_micro_model_library_format.py

```diff
@@ -618,7 +618,6 @@ def test_multiple_relay_modules_aot_graph():
 
     assert os.path.exists(os.path.join(extract_dir, "codegen", "host", "src", "mod1_lib0.c"))
     assert os.path.exists(os.path.join(extract_dir, "codegen", "host", "src", "mod1_lib1.c"))
-    assert os.path.exists(os.path.join(extract_dir, "codegen", "host", "src", "mod1_lib2.c"))
```

Comment:
Could you please comment why `mod2` is not generated anymore?

Reply:
The get_c_metadata() function was being added to the graph executor generated source files but never used.